lib/caesar.rb

```diff
@@ -1,6 +1,40 @@
+
 class Caesar
-  attr_reader :key
+  attr_reader  :word, :shift
+
+  def initialize (word, shift)
+    @word = word
+    @shift = shift
+  end
+
+  def word
+    @word
+  end
 
-  def initialize
+  def eng_to_cipher
```

Comment:
Read the spec more carefully. You will see that `eng_to_cipher` takes two arguments, the text and the key.